panou/src/components/Container/Container.tsx

```diff
@@ -0,0 +1,7 @@
+import styles from "./Container.module.css";
+
+function Container({ children }: any) {
+  return <div className={styles.container}>{children}</div>;
```

Comment:
noi nu folosim asa abordare la css, cand am incercat asa, si nu am vazut plusuri in asta.
dar implementarea ta este de apreciat